.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,22 @@
+<!--
+Please use semantic PR titles that respect this format:
+
+<type>(#<issue number>): <subject>
+
+Quick example:
+
+feat(#1234): add hat wobble
+^--^(#^--^) ^------------^
+|     |     |
+|     |     + - > subject
+|     |
+|     + ------- > issue number
+|
++ ------------- > type: chore, feat, fix, perf, style, test, etc.
```

Comment:
Nice formatting :slightly_smiling_face:

Reply:
I'd hate to take credit for someone else's thing: https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716